app/ui-master/client/loading/loading.css

```diff
@@ -0,0 +1,52 @@
+.loading-animation {
+	position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+
+	display: flex;
+
+	text-align: center;
+	align-items: center;
+	justify-content: center;
+}
+
+.loading-animation > .bounce {
+	display: inline-block;
+
+	width: 10px;
+	height: 10px;
```

Comment:
Is it possible to make this size relative to the parent's height?

Reply:
<3 in the future ;)